client/signup/jetpack-connect/install-step.jsx

```diff
@@ -15,83 +16,100 @@ import JetpackExampleConnect from './example-components/jetpack-connect';
 
 const NEW_INSTRUCTIONS_JETPACK_VERSION = '4.2.0';
 
-export default React.createClass( {
-	displayName: 'JetpackInstallStep',
+class JetpackInstallStep extends Component {
 
-	confirmJetpackInstalled( event ) {
+	static propTypes = {
+		currentUrl: PropTypes.string.isRequired,
```

Comment:
In that case I think we can freely make this prop non-required and set its default prop value as an empty string. What do you think?